### DISCUSSION TITLE
Which File to Use to Modify Aspect Ratios in Comfy Mode?

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [metercai/SimpleSDXL#70] [Q&A] [answered]
What is the proper way to modify the aspect ratios when operating in Comfy mode? I do not want to hack code to do this, but so far I have not found a config text file for Comfy mode.

**Comment by metercai** [comment, marked as answer]
No configuration text. It is in `modules/flags.py`

**Comment by DavidDragonsage** [comment]
Thank you. I will not overwrite flags.py because I suspect it will be overwritten by the next update.
